.github/workflows/validate-renovate.yml

```
@@ -0,0 +1,31 @@
+name: 'Validate renovate config'
+
+on:
+  push:
+    branches:
+      - 'master'
```

This should be `main`, right?
```suggestion
      - 'main'
```